src/ray/protobuf/public/events_node_lifecycle_event.proto

```diff
@@ -26,6 +26,7 @@ message NodeLifecycleEvent {
   enum State {
     ALIVE = 0;
     DEAD = 1;
+    ALIVE_DRAINING = 2;
```

Comment:
I'd suggest that we replace `ALIVE` with `RUNNING`, `DRAINING` etc. The currently one is confusing since ALIVE actually means ALIVE_NO_DRAINING.